Improve postgres upgrade doc (#4010)
GeotrekCE/Geotrek-admin: docs/install/upgrade.rst

::

    pg_restore -h 127.0.0.1 -p 5433 -U <your geotrek user> -d <your geotrek database> ./backup.dump

.. note::

>     Note you should use -h 127.0.0.1 because to keep imported elements owning, you should use geotrek user which is not allowed to connect with default unix socket.

Reviewer: ```suggestion
    Note you have to use `-h 127.0.0.1` to connect with the `geotrek` user (this user cannot connect with the default unix socket). Connecting with `geotrek` is important for restored entities to have the right owner.
```

    Some errors can occurs, around extensions creation or spatial_ref_sys table content.

>     This is normal. We already create this extensions on previous steps

Author: ```suggestion
    This is normal. We already create this extensions on previous steps.
```

>     This is normal. We already create this extensions on previous steps

Author: ```suggestion
    This is normal. We already create these extensions on previous steps
```

>     This is normal. We already create this extensions on previous steps

Author: ```suggestion
    This is normal. We already create these extensions on previous steps.
```
